locale/fr_FR/LC_MESSAGES/django.po

```diff
@@ -2908,6 +2934,9 @@ msgstr "Aucun compte avec cette adresse email n’a été trouvé."
 msgid "A password reset link sent to %s"
 msgstr "Un lien de réinitialisation a été envoyé à %s."
 
+#~ msgid "Remove <a href=\"%(path)s\">%(name)s</a>"
+#~ msgstr "Supprimer <a href=\"%(path)s\">%(name)s</a>"
+
```

Comment:
That seems to be a misplaced duplicate of a [previous string](https://github.com/bookwyrm-social/bookwyrm/pull/1162/files#diff-9c2a068d37359d2eb6e3993c59028e75a2f4576a75182b8b29c61cdf53469bcdL502).